docs/intellij-quickstart.md

```diff
@@ -28,17 +28,13 @@ Make sure you are using OpenJDK12 as the project SDK:
     <img src="./assets/sdk-12.png"/>
 </p>
 
-Open the Maven tool window, and run `mvn install` in the root project:
-
-<p>
-    <img src="./assets/top-level-modules.png"/>
-</p>
+Open the Maven tool window, and run `mvn install -PdevSetup` in the root project:
```

Comment:
Since we are doing the dev build day in and day out, should we make this default, and let any other environment to explicitly pick up whatever profile that needs to be?